data/incipias/first contact.txt

```diff
@@ -0,0 +1,300 @@
+			`	You have to wait a longer period of time until he comes back. "I have good news for you. We believe you that your intentions are only this peaceful ones. We will help you if you fulfill the following condition: Bring us one other proof of a peaceful contact you made to another species."`
+				goto offer
+			label early
+			`	You explain to the Hai that you think it is best to make contact to a species as soon as possible. Because at some point in the distant future there will be contact with them anyways so it is now the best time to establish peaceful connections. After the Hai hears that he stands up and says "I will talk to some people about the things you told me. Please wait here." and leaves the room.`
+			`	A long time passes before the Hai comes back. When he comes back he sits down and says "We are mostly sure now that you have no bad intentions. However, before we start helping you with this matter we want something from you that proves you made peaceful contact to another species. If you can get this for us we will see how we can help you.`
+			label offer
+			`	You ask the Hai what this proof could look like. The Hai answers "That is something I have no answer for. Maybe it would be best if you talk to the government of the species. It was a pleasure to talk to you. I will see you when you have the proof. Oh, and one last thing: It might be better to not tell this species exactly why you need this." With these words he guides you to the exit and you say goodbye to him.`
+				accept
+	to complete
+		or
+			has "wanderer peace proof: done"
+			has "coalition peace proof: offered"
+	on complete
+		conversation
+			branch species
+				has "wanderer peace proof: done"
+			`	You take the box with the gem inside and decide to show the Hai this very unique proof of a good relationship. When you get to the embassy the Hai there already know you and guide you to the person you spoke to last time. "Hello Captain <first> <last>, nice to see you again. If you got back here this means you have a proof?" "Yes, I have something very unique.", you answer. The Hai looks very interested and you pull out the box and put it on the table. "Well", he says, "I hope it is not only this box. It does not seem made by humans, but I fear this wont be enough for a proof." "Of course not", you say and open the box towards the Hai. Suddenly he freezes and just stares at the small gem inside of it. "Indeed, that is nothing made by humans and no Hai could create this either. This must be the work of a really advanced species. Do not worry, I will not ask you from which species you got this. This is more than enough for a proof."`
```

Comment:
Wouldn't the Hai know about your contact with the Wanderers, considering their role in the Wanderer Campaign? In addition, having other options for a peaceful contact (Coalition, Gegno) would be interesting.

Reply:
coalition is an option, I did not do the gegno yet as I wrote this before they were merged, I will see what to do about them